core/domain/fs_services.py

```diff
@@ -21,6 +21,14 @@
 
 
 def get_exploration_file_system_class():
+	""" 
+	Returns:
+		If the development mode of constants is on, the Exploration File System
+		(a datastore-backed read-write file system for a single exploration) is returned.
+
+		Otherwise, it returns Gcs File System (wrapper for a file system based on Google 
+		Cloud Service)
```

Comment:
Indent this block by just 4 spaces and not by 8 spaces. Also, mention the data type in the `Return:` section as you've done in the other file.